src/adapters/local-cache-adapter/index.ts

```diff
@@ -14,9 +14,8 @@ export class LocalCacheAdapter implements ICacheAdapter {
     }
 
     public async set(key: string, values: object, msInterval?: number): Promise<void> {
-        const valuesJson = msInterval
-            ? _.assign({ data: values }, { expirationDate: new Date().getTime() + msInterval })
-            : { data: values };
+        const valuesJson =
```

Comment:
jako podle me ale vo tam vidim tak ti to stejne nevyexpiruje ne? by to chtelo jeste neco co by tu pamet obcas proslo a odmazalo to 

jako mozna ale edge case nebo neni?

Reply:
problém je v tom když nastavíš expiraci na hodnotu 0. ta podmínka jak byla původně, tak pro hodnotu 0 nenastavila exipraci vůbec žádnou. ale já potřebuju pro specifickej případ aby to expirovalo vždy.